scripts/st2bootstrap-el8.template.sh

```diff
@@ -371,6 +359,13 @@ configure_st2chatops() {
 }
 
 trap 'fail' EXIT
+
+# check for RHEL 8
+ELMAJVER=$(cat /etc/redhat-release | sed 's/[^0-9.]*\([0-9.]\).*/\1/')
```

Comment:
Why do we need to check for specific `8` version considering we already running `st2bootstrap-el8` script?
This part looks redundant. I think `RHEL` check if sufficient for the logic below.

Reply:
I agree. Pushing update

Reply:
correct, we don't. Since the base script does this. Removed

Reply:
The old version with 8 version check is still in the codebase. Needs updating.

Reply:
I removed this now. Thanks. I also moved the RHEL check to a function